### ISSUE TITLE
ERROR: dependency 'RandomFields' is not available for package 'wheretowork'

### ISSUE THREAD

**Issue by DanWismer** [NCC-CNC/wheretowork#208]
When installing wheretowork:

if (!require(remotes)) install.packages("remotes")
remotes::install_github("NCC-CNC/wheretowork")

I get an error saying: 
ERROR: dependency 'RandomFields' is not available for package 'wheretowork'

It looks like RandomFields has been removed from CRAN?
https://cran.r-project.org/web/packages/RandomFields/index.html

**Comment by ricschuster** [comment]
It has been yes. I think that's the second time within a year. Might be worth considering ditching the `RandomFields` package

**Comment by jeffreyhanson** [comment]
Yeah, RandomFields was archived, but I'm guessing that it will be back on CRAN soon enough. Yeah, it might be worth dropping RandomFields. It's used in the code for simulating stuff. Do you have any ideas for replacements? Another option could be to make it a soft dependency?

**Comment by DanWismer** [comment]
An idea Jeff and I discussed is either making RandomFields a soft dependency, or even better, adding RandomFields in the DESCRIPTION Remotes: and then updating the renv.lock file

**Comment by ricschuster** [comment]
Soft dependency sounds good to me.

**Comment by DanWismer** [comment]
@jeffreyhanson I have created a new local branch (issue208) and was wondering if you could comment on my approach:
1. Move RandomFields (>= 3.3.8) to the Suggests
2. Find all test_that() calls that have RandomFields and insert `skip_if_not_installed("RandomFields")` before the test_that() call

**Comment by jeffreyhanson** [comment]
Sure  - could you please create a PR for it? Those changes sounds great! In addition to those, I suggest updating the all the functions that depend on RandomFields to check for this dependency and throw an error message if its missing, e.g., something like:

```
assertthat::assert_that(
  requireNamspace("RandomFields"), 
  msg = "please install the \"RandomFields\" package"
)
```
